bitsandbytes/functional.py

```diff
@@ -2592,7 +2631,15 @@ def double_quant(A, col_stats=None, row_stats=None, out_col=None, out_row=None,
     return out_row, out_col, row_stats, col_stats, coo_tensor
 
 
-def transform(A, to_order, from_order="row", out=None, transpose=False, state=None, ld=None):
+def transform(
+    A: torch.Tensor,
+    to_order: TransformOrder,
+    from_order: TransformOrder = "row",
+    out: Optional[torch.Tensor] = None,
+    transpose=False,
+    state: Optional[Tuple[torch.Size, TransformOrder]] = None,
+    ld=None,
```

Comment:
Type?

Reply:
I'm not sure if `ld` belongs here at all considering it's not used, but it is in `nvidia_transform`